concordion/src/main/java/eu/drus/jpa/unit/api/concordion/JpaUnitConcordionRunner.java

```diff
@@ -18,13 +29,14 @@
 
 import eu.drus.jpa.unit.api.JpaUnitException;
 import eu.drus.jpa.unit.concordion.ConcordionInterceptor;
+import eu.drus.jpa.unit.concordion.EnhancedProxy;
 import eu.drus.jpa.unit.concordion.EqualsInterceptor;
 import eu.drus.jpa.unit.concordion.JpaUnitFixture;
 import eu.drus.jpa.unit.spi.DecoratorExecutor;
-import net.sf.cglib.proxy.Callback;
-import net.sf.cglib.proxy.CallbackHelper;
-import net.sf.cglib.proxy.Enhancer;
-import net.sf.cglib.proxy.Factory;
+import net.bytebuddy.ByteBuddy;
+import net.bytebuddy.dynamic.loading.ClassLoadingStrategy;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.bytebuddy.dynamic.loading.ClassLoadingStrategy'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)